source/common/common/utility.h

```diff
@@ -154,6 +154,9 @@ class StringUtil {
 
   /**
    * Convert a string to an unsigned long, checking for error.
+   *
+   * Consider absl::SimpleAtoi instead if using base 10.
+   *
    * @param return true if successful, false otherwise.
    */
   static bool atoull(const char* str, uint64_t& out, int base = 10);
```

Comment:
They are all hex conversions.  I haven't dug around for the best option yet to replace them, but I plan to do that next as part of ongoing #6580 work.

I am also planning to eliminate `StringUtil::atoll` since every existing use case of it is base 10 and can just be replaced with `absl::SimpleAtoi`.